# Generated by Django 4.2.5 on 2023-12-14 10:05

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
import uuid


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('ticket', '0106_event_add_to_sales_event_is_free_event_is_public_and_more'),
    ]

    operations = [
        migrations.AddField(
            model_name='event_transaction_history',
            name='event',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='ticket.event'),
        ),
        migrations.AddField(
            model_name='event_transaction_history',
            name='transaction_status',
            field=models.CharField(choices=[('SUCCESSFUL', 'SUCCESSFUL'), ('FAILED', 'FAILED')], default='FAILED'),
        ),
        migrations.AlterField(
            model_name='event_transaction_history',
            name='ticket',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='ticket.ticket'),
        ),
        migrations.AlterField(
            model_name='event_transaction_history',
            name='user',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
        ),
        migrations.AlterField(
            model_name='ticket',
            name='ticket_key',
            field=models.UUIDField(default=uuid.UUID('7c206553-bac5-4c19-8b6d-bbd4efce4215')),
        ),
        migrations.AlterField(
            model_name='ticket_issues',
            name='error_key',
            field=models.UUIDField(default=uuid.UUID('7ee7727c-32c8-475e-8b8a-a088a1493cc9')),
        ),
        migrations.AlterField(
            model_name='ticket_payment',
            name='payment_id',
            field=models.UUIDField(default=uuid.UUID('f7532c2f-52cd-4576-93b9-10dfa47e8422')),
        ),
    ]
